guides/common/modules/con_ansible-variables-and-parameter-hierarchy.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Ansible_Variables_and_Parameter_Hierarchy_{context}"]
+= Ansible Variables and Parameter Hierarchy
+
+There are different levels on how to set Ansible variables and parameters for a service/host.
```

Comment:
This is a nitpick, off-topic, and a non blocker but something that crushes my technical writer heart. I've literally no idea what we're talking about. Can we stop equating service and host and just say host or service and host or service or host if they are different thing? I feel this is a weak spot.

Reply:
Removed `service/`.